src/canvas/renderCanvas.js

```diff
@@ -213,8 +211,9 @@ export const renderReact = ({
   }
 };
 
-import type { PanAndZoomState } from '../usePanAndZoom';
+import type { PanAndZoomState } from '../util/usePanAndZoom';
```

Comment:
These imports should be at the top of the file. We should add a linter asap to catch these things automatically (I've opened #16)

Reply:
Agreed, we can update issues like these in a PR that adds linter and fixes said linter errors. For now, let's leave it in.

Reply:
I think we should fix this here, because a random import in the middle of a file is very surprising. We can fix less obvious issues later with the linter's help but since this is new code we should try to keep a certain standard haha.

Reply:
Alright, done! (Also did an in browser commit, don't @ me)